exammode/models.py

```diff
@@ -0,0 +1,107 @@
+from django.db import models
+from django.urls import reverse
+from django.utils import timezone
+
+from course.models import CourseInstance, CourseModule
+from exercise.exercise_models import LearningObject
+from userprofile.models import UserProfile
+
+# Create your models here.
+
+
+class ActiveExamSessionManager(models.Manager):
+
+    def get_queryset(self):
+        initial_queryset = super().get_queryset()
+        queryset = [q for q in initial_queryset if (
+            q.can_start <= timezone.now())]
+
+        return queryset
+
+    def is_active(self):
+        initial_queryset = super().get_queryset()
+        queryset = [q for q in initial_queryset if (q.can_start <= timezone.now() and (
```

Comment:
Same as above. Can't this filtering use the filter method instead of a loop?

Reply:
This method is removed while the `get_queryset` method returns a queryset that this method returned as a list.